TU20Bot/Handler.cs

```diff
@@ -19,6 +20,11 @@ public class Handler {
         private readonly CommandService commands;
         private readonly ServiceProvider services;
 
+        private string[] msgArr = { "Hello there!", "Whats poppin",
```

Comment:
```suggestion
        private string[] greetings = {
            "Hello there!",
            "Whats poppin",
            "Wagwan",
            ...
```

 - I try not to use acronyms in my names too much.
 - Stuff like `arr` is describing the type, unlikely we need this in the name :D 
 - Give each entry on long lists their own line